featuremanagement/aio/_featuremanager.py

```diff
@@ -181,7 +189,14 @@ async def is_enabled(self, feature_flag_id, *args, **kwargs):
         :rtype: bool
         """
         targeting_context = self._build_targeting_context(args)
-        return (await self._check_feature(feature_flag_id, targeting_context, **kwargs)).enabled
+        result = await self._check_feature(feature_flag_id, targeting_context, **kwargs)
+        if self._on_feature_evaluated and result.feature.telemetry.enabled:
+            result.user = kwargs.get("user", "")
```

Comment:
Here is different from the implementation in async feature manager where you set result.user = targeting_context.user_id

Reply:
@zhiyuanliang-ms, something went wrong with the merge. If you look at the last few commits I've been trying to fix them. But thanks for pointing this one out.